docs/usage/configuration-options.md

```diff
@@ -2138,6 +2138,22 @@ Here's an example of how you would define PR priority so that devDependencies ar
 
 The PR title is important for some of Renovate's matching algorithms (e.g. determining whether to recreate a PR or not) so ideally don't modify it much.
 
+## prettyNewMajor
+
+This option allows editing `newMajor` value by templating, it is defaulted to prepend `v` to the `newMajor` value.
+
+For example, to create a PR title with the changed version:
+
+`{ "commitMessageExtra": "{{prettyNewMajor}}" }`
+
+## prettyVersion
```

Comment:
These are not config options

Reply:
Can you please elaborate more where you think they will they fit in ? Should they be removed from documentation or they shouldn't be assignable configuration ?

Reply:
> Exposed a handlebars function to enable prettifying `newVersion` and `newMajor`.

It looks like you're adding "Handlebars helpers" in this PR.

> Can you please elaborate more where you think they will they fit in ?

I think the [Renovate docs, templates, additional Handlebars helpers](https://docs.renovatebot.com/templates/#additional-handlebars-helpers) is the correct place to put this information.

> Should they be removed from documentation or they shouldn't be assignable configuration ?

They shouldn't be assignable configuration, it looks like these are "Handlebars helpers".